### ISSUE TITLE
Deleting Just One Item

### ISSUE THREAD

**Issue by Jisenku** [SimonDEvans/notes#4]
It would be very nice to have that feature instead of deleting it all

**Comment by fifn2** [comment]
Yes, this seems like a pretty necessary feature even for a simple Notes app.
